runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -95,7 +95,7 @@ public AddressSpaceView(@Nonnull final CorfuRuntime runtime) {
             cacheBuilder.weigher((k, v) -> (int) (CACHE_KEY_SIZE + MetricsUtils.sizeOf.deepSizeOf(v)));
         }
 
-        if (cacheDisabled) {
+        if (true) {
             cacheBuilder.maximumSize(0); // Do not allocate memory when cache is disabled.
```

Comment:
The code will never go to the else statements?

Reply:
Was testing something. Removed.